app/utils/translationStrings.js

```diff
@@ -0,0 +1,1018 @@
+const translations = {
```

Comment:
I'm a little confused about the need for this file. Can you explain a but about what it's used for?